crates/solidity/testing/snapshots/cst_output/ReceiveFunctionDefinition/simple/generated/0.4.11.yml

```diff
@@ -6,17 +6,14 @@ Source: >
   3  │     // https://github.com/NomicFoundation/slang/issues/359                       │ 65..123
```

Comment:
are we now able to remove this comment from source?

```solidity
fallback () { }
receive

Reply:
done